mkt/webapps/migrations/0001_initial.py

```diff
@@ -0,0 +1,578 @@
+            bases=(caching.base.CachingMixin, models.Model),
+        ),
+        migrations.CreateModel(
+            name='BlockedSlug',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('modified', models.DateTimeField(auto_now=True)),
+                ('name', models.CharField(default=b'', unique=True, max_length=255)),
+            ],
+            options={
+                'db_table': 'addons_blocked_slug',
+            },
+            bases=(caching.base.CachingMixin, models.Model),
+        ),
+        migrations.CreateModel(
+            name='ContentRating',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('modified', models.DateTimeField(auto_now=True)),
+                ('ratings_body', models.PositiveIntegerField(choices=[(0, b'CLASSIND'), (1, 'Generic'), (2, b'USK'), (3, b'ESRB'), (4, b'PEGI')])),
+                ('rating', models.PositiveIntegerField()),
+            ],
+            options={
+                'db_table': 'webapps_contentrating',
+            },
+            bases=(caching.base.CachingMixin, models.Model),
+        ),
+        migrations.CreateModel(
+            name='Geodata',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('modified', models.DateTimeField(auto_now=True)),
+                ('restricted', models.BooleanField(default=False)),
+                ('popular_region', models.CharField(max_length=10, null=True)),
+                ('banner_regions', mkt.webapps.models.RegionListField(null=True)),
+                ('region_de_usk_exclude', models.BooleanField(default=False)),
+                ('region_cn_status', models.PositiveIntegerField(default=2, help_text=b'<django.utils.functional.__proxy__ object at 0x2b32d90> approval status', db_index=True, choices=[(0, 'Incomplete'), (16, 'Unlisted'), (2, 'Pending approval'), (4, 'Published'), (5, 'Banned from Marketplace'), (11, 'Deleted'), (12, 'Rejected'), (13, 'Approved but private'), (15, 'Blocked')])),
+                ('region_cn_nominated', models.DateTimeField(help_text=b'<django.utils.functional.__proxy__ object at 0x2b32d90> nomination date', null=True)),
+                ('region_br_iarc_exclude', models.BooleanField(default=False)),
+                ('region_de_iarc_exclude', models.BooleanField(default=False)),
+                (b'TranslatedField', mkt.translations.fields.TranslatedField(null=True, on_delete=django.db.models.deletion.SET_NULL, to_field=b'id', to=mkt.translations.models.Translation, blank=True, unique=True)),
```

Comment:
Huh. Maybe because I didn't return a unicode string from `reconstruct()` on that field. Probably some python3-compat nonsense.